browser/caliban.py

```diff
@@ -699,63 +684,30 @@ def create_cell_info(self, feature):
             self.cell_info[feature][cell]['frames'] = []
 
             for frame in range(self.annotated.shape[0]):
-                if cell in annotated[frame, :, :]:
+                if cell in annotated[frame, ...]:
                     self.cell_info[feature][cell]['frames'].append(int(frame))
             self.cell_info[feature][cell]['slices'] = ''
 
         self.info_changed = True
 
-    def create_lineage(self):
```

Comment:
`create_lineage` is used to save npzs as trk files. currently not in use for browser version so it's fine to take out in this PR, but I think we'll add it back in to make a shared backend for browser/desktop at some point

Reply:
Yeah, we can definitely add it back in the future but it did not run in its current state.